dev-requirements.txt

```diff
@@ -0,0 +1,16 @@
+APScheduler==3.5.1
+awsebcli==3.19.0
+blinker==1.4
+coverage==5.3
+Flask==1.0.2
+Flask-JWT-Extended==3.24.1
+Flask-Mail==0.9.1
+Flask-Migrate==2.5.3
+flask-restx==0.2.0
+Flask-Testing==0.8.0
+gunicorn==20.0.4
+psycopg2-binary==2.8.6
+PyMySQL==0.10.1
+python-dotenv==0.14.0
+regex==2020.7.14
```

Comment:
where do we use this?

Reply:
Ignore this comment if you are asking the question specifically from the POV of dev-requirements.
If you are asking in general then:-
`psycopg2-binary` for postgresql locally
`PyMySQL` for MySQL
`python-dotenv` for exporting env variables from .env file automatically
`regex` -- NO IDEA

Reply:
HI, @SanketDG regex is used to check the validity of name, email and username fields.

Reply:
Sorry I was just talking about the highlighted line haha, but thank you for the explanation @PrashanthPuneriya